.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  "extends": "airbnb-base",
+  "env": {
+    "browser": true,
+    "mocha": true,
```

Comment:
do we even use mocha?

Reply:
No we don't. I'll get it removed. We need to do the same with the other apps too